invenio_mail/ext.py

```diff
@@ -67,5 +67,11 @@ def init_config(self, app):
     def print_email(self, message, app):
         """Print mail to stream."""
         with self._lock:
-            self.stream.write("{0}\n{1}\n".format(message.as_string(), '-'*79))
-            self.stream.flush()
+            invenio_mail = app.extensions['invenio-mail']
+            invenio_mail.stream.write(
+                "{0}\n{1}\n".format(message.as_string(), '-'*79))
+            invenio_mail.stream.flush()
+
+    def send_email(self, message):
```

Comment:
Yes, if you want to use the celery task to send emails, I think the module should explicit import it. Otherwise you simply use Message().send() from Flask-Mail.